### DISCUSSION TITLE
How to set Algolia search to show localized results?

### DISCUSSION THREAD

**Discussion by GasimGasimzada** [reactjs/reactjs.org-translation#250] [Team Posts]
Currently, searching in Algolia shows English results that link to english pages. Is there a way to configure it; so that, it shows localized results that link to localized pages?

**Comment by Fcmam5** [comment]
Create a [new Algolia application](https://community.algolia.com/docsearch/#join-docsearch-program) for for the language you're working on and then replace the apiKey in `src/components/LayoutHeader/DocSearch.js` .

Example PR: https://github.com/reactjs/ar.reactjs.org/pull/128/files
